libevmasm/Instruction.h

```diff
@@ -267,28 +268,28 @@ inline unsigned getLogNumber(Instruction _inst)
 /// @returns the PUSH<_number> instruction
 inline Instruction pushInstruction(unsigned _number)
 {
-	assertThrow(_number <= 32, InvalidOpcode, std::string("Invalid PUSH instruction requested (") + std::to_string(_number) + ").");
+	solAssert(_number <= 32);
```

Comment:
Note that this validation was impossible to trigger via assembly import, because the import blindly converts the value to `u256` without checking the range. The conversion silently truncates the value to 256 bits so it can never exceed 32 bytes. This means that this case is not validated at all.

This is actually a bug in assembly import - out-of-range PUSH values result in something wrong being pushed to the stack.